include/battle.h

```diff
@@ -126,34 +127,34 @@ typedef struct UnkBtlCtxSub_76 {
 typedef struct UnkBattlemonSub {
```

Comment:
can this struct be named? it seems to be about moves that affect the battle status

Reply:
I'm very open to suggestions if you have any

Reply:
BattleMonMoveStatus maybe?

Reply:
ehhh it also includes the flags for custap and quick claw triggers, but those might arguably be related to moves? it's less move statuses though and more just flags, counts, etc